pkg/ui/src/app.tsx

```diff
@@ -54,6 +54,7 @@ import { ConnectedDecommissionedNodeHistory } from "src/views/reports";
 
 import "nvd3/build/nv.d3.min.css";
 import "react-select/dist/react-select.css";
+import "antd/es/tooltip/style/css";
```

Comment:
Why is this here?

Reply:
@dhartunian 
1. Badges styles are not 100% ready to be honest, I picked this component as easiest one for demo purposes only. It has the same colors because not all styles are defined.
It is already done in master.

2. Component width in Firefox. It is really a problem - **nib** library (extension for stylus) does not add vendor prefixes for **width** attribute with `max-content` or `fit-content` values.
I will create separate issue on this because it doesn't relate to Storybook integration

3. "antd/es/tooltip/style/css" this is a global style on which we depend. Previously it was imported in Tooltip component but was applied globally. To make it explicit that it is global styles I put it in top level component.

Reply:
3. (Updated) you're right, it's unnecessary to move styles import to App.tsx. Reverted this change back.